### ISSUE TITLE
Fails to work with Django 1.8

### ISSUE THREAD

**Issue by nijel** [omab/python-social-auth#526]
With Django 1.8a1 psa fails, though I'm not sure if that's fixable here:

```
Traceback (most recent call last):
  File "./manage.py", line 31, in <module>
    execute_from_command_line(sys.argv)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute_from_command_line
    utility.execute()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/core/management/__init__.py", line 304, in execute
    django.setup()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/registry.py", line 82, in populate
    app_config = AppConfig.create(entry)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/opt/python/2.7.9/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/social/apps/django_app/__init__.py", line 14, in <module>
    from social.apps.django_app.utils import load_strategy
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/social/apps/django_app/utils.py", line 20, in <module>
    Strategy = module_member(STRATEGY)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/social/utils.py", line 23, in module_member
    module = import_module(mod)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/social/utils.py", line 17, in import_module
    __import__(name)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/social/strategies/django_strategy.py", line 4, in <module>
    from django.contrib.contenttypes.models import ContentType
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/contrib/contenttypes/models.py", line 159, in <module>
    class ContentType(models.Model):
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/db/models/base.py", line 92, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/registry.py", line 230, in get_containing_app_config
    self.check_apps_ready()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/apps/registry.py", line 121, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

**Comment by DanielJDufour** [comment]
Hi, Nijel.  I wrote a fix, which you can find here: https://github.com/DanielJDufour/python-social-auth/commit/b51629fcc54ad755f1b17f27712aacc860f34129.

However, I'm not a core developer, so I can't really say if this fix follows best practices.  Basically, I moved stuff from the __init__ file in django_app to a ready method in the PythonSocialAuthConfig class that I created.  See this for more information and why I did this: https://docs.djangoproject.com/en/1.7/ref/applications/

Hope this helps and good luck! :)

**Comment by omab** [comment]
This was fixed once merged #550
